frontend/src/concepts/areas/const.ts

```diff
@@ -58,4 +58,9 @@ export const SupportedAreasStateMap: SupportedAreasState = {
     requiredComponents: [StackComponent.TRUSTY_AI],
     reliantAreas: [SupportedArea.BIAS_METRICS],
   },
+  [SupportedArea.DISTRIBUTED_WORKLOADS]: {
+    featureFlags: [],
+    // featureFlags: ['disableDistributedWorkloads'], // TODO mturley put these back and troubleshoot
+    // requiredComponents: [StackComponent.KUEUE], // TODO mturley put these back and troubleshoot
```

Comment:
Are you unsure how to set this up?

Reply:
I thought I had it figured out, but with just the `featureFlags` line here uncommented (not `requiredComponents`, I didn't have the component enabled yet on the cluster I was testing on) it wasn't passing the area available check for some reason even though the flag was false in the config. I was going to ask you about this when I got back to it.

Reply:
Ah okay, wasn't sure why it was commented. Yeah, it looks good 👍

Reply:
If I put back `featureFlags: ['disableDistributedWorkloads']` here though, it behaves as if the feature is disabled when it's not. Do you know what might be going on there?

Reply:
You need to deploy your CRD to the cluster -- the field is forbidden in the CR without the CRD change and thus defaults to disabled.

Reply:
ahhh okay. i thought the dev server would use the mock cluster config where I added it, but I guess that's just for tests.